### ISSUE TITLE
14x Unity 2022.2 Fullscreen Templates

### ISSUE THREAD

**Issue by Dawie3565** [AmplifyCreations/AmplifyShaderEditor-Feedback#114]
HDRP 14.0 introduces a new material type in ShaderGraph to create fullscreen effects. Shaders of the fullscreen type can be used in fullscreen custom passes, custom post processes and C# scripting.

Fullscreen - custom passes
Fullscreen - custom post processes

**Comment by Dawie3565** [comment]
**Fullscreen - custom passes**
https://github.com/Unity-Technologies/Graphics/blob/86cdbd182b8fa8aeda2ff536434f9456f3e5029b/Packages/com.unity.render-pipelines.high-definition/Documentation~/Custom-Pass-Creating.md 

**Fullscreen - custom post processes**
https://github.com/Unity-Technologies/Graphics/blob/86cdbd182b8fa8aeda2ff536434f9456f3e5029b/Packages/com.unity.render-pipelines.high-definition/Documentation~/Custom-Post-Process.md

Note they have a 3rd one but no documentation found yet ?
**Fullscreen - Custom render target**

**Comment by LazyEti** [comment]
I've been coming back to this tracker for more than a year and it has been mentioned a couple of time on discord that it was planned to be added in the near future. But at this rate I'm wondering if this feature is around the corner or if I should go down the rabbit hole and create my own custom template.
@Dawie3565 Is there an ETA on this / Is there a chance that this one could get bumped to a higher priority??

**Comment by MCxYY** [comment]
@Dawie3565 I really need this feature for custom post-processing, so I hope to support it as soon as possible

**Comment by LazyEti** [comment]
@MCxYY 
I don't think this template will get added any time soon but until it is, I use a dirty work around:

1. Create a new unlit, transparent shader. Ex: http://paste.amplify.pt/view/raw/5be301a0
2. Remove all additional options. The only pass that you want to keep in the sub-shader is the Forward pass
3. Create a new Screen Space Overlay Canvas.
4. Add an image child that fills the screen.
5. Then just assign your shader to the fill image.

If you want the effect to render before post processing / transparents :
- Use a screen Space Camera canvas instead and link your main cam.
- Set the plane distance to 1 and lower the "order in layer"

 It's definitely not the best workflow and i haven't looked into performances but it does seem to work okay for now

**Comment by MCxYY** [comment]
@LazyEti I tried it and it works!  thx!!!

**Comment by Dawie3565** [comment]
Note 
Notify Discord User **domfada** when testing new templates 
domfada has been making custom templates for this in ASE and may have feedback and lessons learn for improvement / mistakes found
